pigar/utils.py

```diff
@@ -81,13 +81,29 @@ def print_table(rows, headers=['PACKAGE', 'CURRENT', 'LATEST']):
 
 
 def parse_reqs(fpath):
-    pkg_v_re = re.compile(r'^(?P<pkg>[^><==]+)[><==]{,2}(?P<version>.*)$')
     """Parse requirements file."""
+    print(
+        Color.BLUE("Reading requirements from " + fpath)
+    )
```

Comment:
The util module shouldn't have this kind of logs, you could think of it as a side effect.

Reply:
@damnever In that case, maybe the function could return a list of all the fpaths in addition to the requirements dictionary, and the calling function in __main__.py could actually print the logs. Would that be better?

Reply:
Remove it could be just fine for now.

Reply:
@damnever Ok, I will remove it.